app/ui-angular/src/app/store/integration/integration.service.ts

```diff
@@ -23,6 +23,10 @@ function getFlowsCount(integration: Integration) {
 }
 
 function transform(integration: Integration): Integration {
+  const flows = Object.getOwnPropertyDescriptor(integration, 'flows');
```

Comment:
Knowing this codebase it's probably a better and more simple fix fix to just look and see if the value is in there or not and just add it, i.e.:

```
if (!'flows' in integration) {
   integration.flows = []
}
```

Same goes for `tags` BTW if that's a problem, since there's another blind check further down in this file.